src/FsAutoComplete.Core/ParseAndCheckResults.fs

```diff
@@ -770,4 +770,4 @@ type ParseAndCheckResults
   member __.GetAST = parseResults.ParseTree
   member __.GetCheckResults: FSharpCheckFileResults = checkResults
   member __.GetParseResults: FSharpParseFileResults = parseResults
-  member __.FileName: string<LocalPath> = UMX.tag parseResults.FileName
+  member __.FileName: string<LocalPath> = Utils.normalizePath parseResults.FileName
```

Comment:
Does this mean we had cases where non-normalized paths were leaking through the system that we just blindly tagged? Essentially, should I look at all of these swaps from `UMX.tag` to `Utils.normalizePath` as gaps/bugs that you found and squashed?

Reply:
Yeah I discovered some bugs with this and I was a bit ham-fisted here with fixing these places. Though I think the transparent compiler might be missing some calls to FileSystem.NormalizePath